integration_test/third_party_apps_test.go

```diff
@@ -641,6 +641,19 @@ func runSingleTest(ctx context.Context, logger *logging.DirectoryLogger, vm *gce
 	if metadata.ExpectedMetrics != nil {
 		logger.ToMainLog().Println("found expectedMetrics, running metrics test cases...")
 
+		// All integrations are expected to set the instrumentation_source label.
```

Comment:
Is it possible to add an integration test for the non-3rd party app receivers to verify the behavior of the instrumentation scope labels there?

Reply:
What are you including in "non-3rd party app receivers"? The existing integration test should be checking the agent metrics, and GMP is separate from this, so isn't everything else being covered by `third_party_apps_test`?

Reply:
@qingling128 do you mean to check that the instrumentation scope labels are not present in the system metrics? To make sure consumers of those metrics see no discontinuity with their timeseries once this lands?

Reply:
Yes. Basically how the labels behave for the OTLP receiver and host metrics receivers. Ok N the documentation we probably wanna call out their difference as well.

Reply:
Since system metrics' descriptors are pre-defined, there is no way for Ops Agent to report additional metric labels, so we don't have to worry that users will see any changes. (At worst, the points would just be rejected - which the existing integration tests should catch.)